model/context.go

```diff
@@ -107,10 +107,37 @@ func truncate(s string) string {
 	return s
 }
 
+type NetworkInfo struct {
+	EffectiveType *string
+	RoundTripTime *int64
+	Downlink      *int64
+	DownlinkMax   *int64
+	SaveData      *bool
+	Type          *string
+}
+
+func (ni *NetworkInfo) fields() common.MapStr {
+	fields := common.MapStr{}
+	if ni == nil {
+		return fields
+	}
+	utility.Set(fields, "type", ni.Type)
+	utility.Set(fields, "downlink", ni.Downlink)
+	utility.Set(fields, "downlinkMax", ni.DownlinkMax)
+	utility.Set(fields, "effectiveType", ni.EffectiveType)
+	utility.Set(fields, "roundTripTime", ni.RoundTripTime)
```

Comment:
Maybe just "rtt"? It's a pretty well known acronym.

Reply:
maybe is not 100% known (particularly for non native speakers)?  what's the harm in `roundTripTime` ?

Reply:
There's nothing wrong with "roundTripTime"; I'm arguing that it's more common to write "rtt" than it is "round-trip time" in technical settings. It is of course not known by 100% of people, just as the phrase wouldn't be; but it is a standard technical phrase and acronym in our industry: https://en.wikipedia.org/wiki/Round-trip_delay_time.

Looking through beats, I didn't find any fields with "roundTripTime" or "round_trip_time" in the name. There are several occurrences of fields with "rtt" in the name though:
- https://www.elastic.co/guide/en/beats/filebeat/current/exported-fields-googlecloud.html#_vpcflow_2
- https://www.elastic.co/guide/en/beats/heartbeat/current/exported-fields-tls.html#_rtt_6 (repeats for other protocols)